tasks/section1.yml

```diff
@@ -1252,15 +1252,20 @@ print $NF }'"
       - rule_1.7.1.4
 
 - name: "NOTSCORED | 1.7.1.4 | PATCH | Ensure permissions on /etc/motd are configured"
-  command: /bin/true
+  file:
+      dest=/etc/motd
+      state=file
+      owner=root
+      group=root
+      mode=0644
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md